[Documentation] Available queryParam types

lifeart opened the pull request:
For now, only 'boolean' query params type present in documentation.

This pr brings all available types for queryParam in documentation.


https://github.com/emberjs/ember.js/blob/6fc89cdf13124d88b8ae6adf99bb02a8c0cdf508/packages/ember-routing/lib/system/router.js#L732

lifeart committed:
Available queryParam types

lifeart committed:
If query param type not specified, it will be `string`.

btecu commented on packages/ember-routing/lib/ext/controller.js:
```diff
@@ -16,6 +16,8 @@ ControllerMixin.reopen({
     `this.category` and `this.page`.
     By default, Ember coerces query parameter values using `toggleProperty`.
     This behavior may lead to unexpected results.
+    Available queryParam types: `boolean`, `number`, `array`
```
Also `string`?

lifeart replied:
@btecu nope, https://github.com/emberjs/ember.js/blob/6fc89cdf13124d88b8ae6adf99bb02a8c0cdf508/packages/ember-routing/lib/system/router.js#L732

btecu replied:
Isn't `string` the default?

lifeart replied:
it can be `null`, `undefined`, or `string`

lifeart replied:
in happy path - `string`

delusioninabox committed:
Update unless documentation with block example and better clarification

delusioninabox committed:
Make greeting plural to match language around it

locks committed:
Update if-unless.ts

simonihmig committed:
[Feature] Update initializer blueprint for ember-mocha 0.14

Related to #16863

simonihmig committed:
[Feature] Update instance-initializer blueprint for ember-mocha 0.14

Related to #16863

Commit:
[FEAT] Updates decorator implementation to stage 1

sukima committed:
Fix incorrect component-test invocation

Prior to this change, the command `ember g component x-foo --path foo`
was originally producing angle bracket syntax but this seems incorrect
and should be the full nested invocation and curly bracket syntax.

This change fixes that oversight.

Issue found and discussed with @ppcano via Discord.

Commit:
[BUGFIX lts] Enable maximum rerendering limit to be customized.

As of Ember 3.4 the autorun assertion is removed, and usage of native
promises has increase the likely hood that this rerender limit of 10
will be hit in "normal" scenarios.

Co-authored-by: Kris Selden <kris.selden@gmail.com>

dependabot-support committed:
Bump tslint from 5.13.0 to 5.13.1

Bumps [tslint](https://github.com/palantir/tslint) from 5.13.0 to 5.13.1.
- [Release notes](https://github.com/palantir/tslint/releases)
- [Changelog](https://github.com/palantir/tslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/palantir/tslint/compare/5.13.0...5.13.1)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.411.0 to 2.412.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.411.0 to 2.412.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.411.0...v2.412.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump ember-cli-version-checker from 3.0.1 to 3.1.0

Bumps [ember-cli-version-checker](https://github.com/rwjblue/ember-cli-version-checker) from 3.0.1 to 3.1.0.
- [Release notes](https://github.com/rwjblue/ember-cli-version-checker/releases)
- [Changelog](https://github.com/ember-cli/ember-cli-version-checker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rwjblue/ember-cli-version-checker/compare/v3.0.1...v3.1.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17686 from emberjs/dependabot/npm_and_yarn/tslint-5.13.1

dependabot[bot] committed:
Merge pull request #17687 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.412.0

dependabot[bot] committed:
Merge pull request #17688 from emberjs/dependabot/npm_and_yarn/ember-cli-version-checker-3.1.0

sukima committed:
Fix helper blueprint for module unification

Prior to this change, the classic blueprint used a different
construction for helper files. Since module unification this has
changed.

This commit updates the blueprints to generate the new construction code
for helpers when module unification is in use and still offers the
classic construction code when not.

Fixes issue #17556

Commit:
skip action tests that fail in class-loose mode

Commit:
[BUGFIX] Ensure tagForProperty works on class constructors

`tagForProperty` was returning the constant tag for functions, which
meant it would return the constant tag for properties on class
constructors. If a computed property was defined on the class
constructor it would fail because of this.

rwjblue committed:
Merge pull request #17690 from emberjs/change-decorators-to-stage-1

[FEAT] Updates decorator implementation to stage 1

rwjblue committed:
Merge pull request #17685 from sukima/angle-bracket-blueprint

Angle bracket blueprint

rwjblue committed:
Merge pull request #17689 from sukima/helper-blueprints-issue-17556

Fix helper blueprint for module unification

Commit:
Allow runloop to settle via autorun in tests.

stefanpenner committed:
Revert "Revert upgrade to hash-for-dep 1.4.7"

This reverts commit d8cef0d54375bc1a69ad91e2c85e9f9fa1116656.

stefanpenner committed:
Drop resoltns, update lock file, bump hash-for-dep

Commit:
fix isElementDescriptor

rwjblue committed:
Merge pull request #17694 from emberjs/fix-hash-for-dep-issue

Fix hash for dep issue

rwjblue committed:
Merge pull request #17691 from emberjs/bugfix/ensure-tag-for-property-works-on-classes

[BUGFIX] Ensure tagForProperty works on class constructors

dependabot-support committed:
Bump ember-cli-version-checker from 3.1.0 to 3.1.1

Bumps [ember-cli-version-checker](https://github.com/rwjblue/ember-cli-version-checker) from 3.1.0 to 3.1.1.
- [Release notes](https://github.com/rwjblue/ember-cli-version-checker/releases)
- [Changelog](https://github.com/ember-cli/ember-cli-version-checker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rwjblue/ember-cli-version-checker/compare/v3.1.0...v3.1.1)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.412.0 to 2.413.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.412.0 to 2.413.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.412.0...v2.413.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17695 from emberjs/dependabot/npm_and_yarn/ember-cli-version-checker-3.1.1

dependabot-support committed:
Bump broccoli-persistent-filter from 2.2.1 to 2.2.2

Bumps [broccoli-persistent-filter](https://github.com/stefanpenner/broccoli-persistent-filter) from 2.2.1 to 2.2.2.
- [Release notes](https://github.com/stefanpenner/broccoli-persistent-filter/releases)
- [Changelog](https://github.com/stefanpenner/broccoli-persistent-filter/blob/master/CHANGELOG.md)
- [Commits](https://github.com/stefanpenner/broccoli-persistent-filter/compare/v2.2.1...v2.2.2)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump ember-cli from 3.8.0 to 3.8.1

Bumps [ember-cli](https://github.com/ember-cli/ember-cli) from 3.8.0 to 3.8.1.
- [Release notes](https://github.com/ember-cli/ember-cli/releases)
- [Changelog](https://github.com/ember-cli/ember-cli/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ember-cli/ember-cli/compare/v3.8.0...v3.8.1)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17697 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.413.0

dependabot[bot] committed:
Merge pull request #17699 from emberjs/dependabot/npm_and_yarn/broccoli-persistent-filter-2.2.2

dependabot-support committed:
Bump broccoli-uglify-sourcemap from 2.2.0 to 3.0.0

Bumps [broccoli-uglify-sourcemap](https://github.com/ember-cli/broccoli-uglify-sourcemap) from 2.2.0 to 3.0.0.
- [Release notes](https://github.com/ember-cli/broccoli-uglify-sourcemap/releases)
- [Changelog](https://github.com/ember-cli/broccoli-uglify-sourcemap/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ember-cli/broccoli-uglify-sourcemap/compare/v2.2.0...v3.0.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17696 from emberjs/dependabot/npm_and_yarn/ember-cli-3.8.1

bekzod committed:
cleanup `canSetTypeOfInput`

rwjblue committed:
Merge pull request #17683 from rwjblue/failing-test-for-tracked-set-outside-runloop

Remove `hasViews` guard.

rwjblue committed:
Merge pull request #17684 from rwjblue/expose-customizable-rerender-limit

[BUGFIX lts] Enable maximum rerendering limit to be customized.

rwjblue committed:
Merge pull request #17668 from emberjs/dependabot/npm_and_yarn/broccoli-uglify-sourcemap-3.0.0

Bump broccoli-uglify-sourcemap from 2.2.0 to 3.0.0

eflanagan0 committed:
extend from glimmer component for octane blueprint

Fix #17701.

kategengler committed:
Add v3.9.0-beta.3 to CHANGELOG
[ci skip]

(cherry picked from commit b93488b1857afb555b57aecf1a5b6f00a5a8c30f)

dependabot-support committed:
Bump aws-sdk from 2.413.0 to 2.414.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.413.0 to 2.414.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.413.0...v2.414.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump puppeteer from 1.12.2 to 1.13.0

Bumps [puppeteer](https://github.com/GoogleChrome/puppeteer) from 1.12.2 to 1.13.0.
- [Release notes](https://github.com/GoogleChrome/puppeteer/releases)
- [Commits](https://github.com/GoogleChrome/puppeteer/compare/v1.12.2...v1.13.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump backburner.js from 2.5.0 to 2.6.0

Bumps [backburner.js](https://github.com/BackburnerJS/backburner.js) from 2.5.0 to 2.6.0.
- [Release notes](https://github.com/BackburnerJS/backburner.js/releases)
- [Commits](https://github.com/BackburnerJS/backburner.js/compare/v2.5.0...v2.6.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump ember-cli-version-checker from 3.1.1 to 3.1.2

Bumps [ember-cli-version-checker](https://github.com/rwjblue/ember-cli-version-checker) from 3.1.1 to 3.1.2.
- [Release notes](https://github.com/rwjblue/ember-cli-version-checker/releases)
- [Changelog](https://github.com/ember-cli/ember-cli-version-checker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rwjblue/ember-cli-version-checker/compare/v3.1.1...v3.1.2)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17704 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.414.0

dependabot-support committed:
Bump eslint from 5.14.1 to 5.15.1

Bumps [eslint](https://github.com/eslint/eslint) from 5.14.1 to 5.15.1.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v5.14.1...v5.15.1)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17705 from emberjs/dependabot/npm_and_yarn/puppeteer-1.13.0

dependabot[bot] committed:
Merge pull request #17706 from emberjs/dependabot/npm_and_yarn/backburner.js-2.6.0

dependabot[bot] committed:
Merge pull request #17708 from emberjs/dependabot/npm_and_yarn/ember-cli-version-checker-3.1.2

dependabot[bot] committed:
Merge pull request #17707 from emberjs/dependabot/npm_and_yarn/eslint-5.15.1

rwjblue committed:
Merge pull request #17702 from efx/fix-17701

extend from glimmer component for octane blueprint

rwjblue committed:
Merge pull request #17700 from bekzod/cleanup-can-set-type

cleanup `canSetTypeOfInput`

Commit:
[BUGFIX beta] Allow accessors in mixins

This fix allows native object accessors to work in Ember Mixins. It does
this by looping over the properties of a new mixin and extracting their
descriptors, checking to see if any of them are accessors. If they are,
it wraps them in a descriptor decorator.

dependabot-support committed:
Bump aws-sdk from 2.414.0 to 2.415.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.414.0 to 2.415.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.414.0...v2.415.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17711 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.415.0

rwjblue committed:
Merge pull request #17658 from ppcano/mu-dummy-blueprints

Add missing blueprints for MU `dummy` app.

simonihmig committed:
Instance-initializer blueprint: destroy instance before application

See https://github.com/emberjs/ember.js/pull/17412#discussion_r26304

simonihmig committed:
Initializer blueprint: remove setupTest()

As it is not needed, see https://github.com/emberjs/ember.js/pull/17411#discussion_r263048203

ppcano committed:
Fix blueprint helper test

It fixes the test when generating a helper in a MU `dummy` app.

It continues the work of #17658 that requires this change when #17689 was merged.

Turbo87 committed:
Merge pull request #17716 from ppcano/fix-blueprint-helper

Fix blueprint helper test

luxzeitlos committed:
remove init and add types to router

chancancode committed:
[CLEANUP] unflag EMBER_GLIMMER_NAMED_ARGUMENTS

chancancode committed:
[CLEANUP] unflag GLIMMER_CUSTOM_COMPONENT_MANAGER

chancancode committed:
[CLEANUP] unflag EMBER_GLIMMER_ANGLE_BRACKET_INVOCATION

chancancode committed:
[CLEANUP] unflag EMBER_GLIMMER_ARRAY_HELPER

chancancode committed:
[CLEANUP] unflag GLIMMER_MODIFIER_MANAGER

rwjblue committed:
Merge pull request #17713 from simonihmig/blueprint-qunit-init

Initializer blueprint: remove setupTest()

rwjblue committed:
Merge pull request #17710 from emberjs/bugfix/allow-accessors-in-mixins

[BUGFIX beta] Allow accessors in mixins

tomdale committed:
Adds nested lookup in angle bracket invocation syntax

See https://github.com/emberjs/rfcs/pull/457 for RFC.
This feature is added beind the EMBER_GIMMER_ANGLE_BRACKET_NESTED_LOOKUP feature flag.

rwjblue committed:
Merge pull request #17718 from emberjs/cleanup-features

Cleanup features

rwjblue committed:
Merge pull request #17712 from simonihmig/blueprint-qunit-instance-init

Instance-initializer blueprint: destroy instance before application

dependabot-support committed:
Bump aws-sdk from 2.415.0 to 2.417.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.415.0 to 2.417.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.415.0...v2.417.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17720 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.417.0

rwjblue committed:
Merge pull request #17637 from luxferresum/cleanup_router

Cleanup router a tiny little bit

rwjblue committed:
Merge pull request #17412 from simonihmig/instance-init-mocha

[Feature] Update instance-initializer blueprint for ember-mocha 0.14

rwjblue committed:
Merge pull request #17411 from simonihmig/initializer-blueprint-mocha

[Feature] Update initializer blueprint for ember-mocha 0.14

dependabot-support committed:
Bump aws-sdk from 2.417.0 to 2.418.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.417.0 to 2.418.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.417.0...v2.418.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17722 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.418.0

kategengler committed:
Add v3.9.0-beta.4 to CHANGELOG
[ci skip]

(cherry picked from commit fc75c3bd63ad10851bd7731e1d7deb7ecff8a76a)

dependabot-support committed:
Bump aws-sdk from 2.418.0 to 2.419.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.418.0 to 2.419.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.418.0...v2.419.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17723 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.419.0

dependabot-support committed:
Bump tslint from 5.13.1 to 5.14.0

Bumps [tslint](https://github.com/palantir/tslint) from 5.13.1 to 5.14.0.
- [Release notes](https://github.com/palantir/tslint/releases)
- [Changelog](https://github.com/palantir/tslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/palantir/tslint/compare/5.13.1...5.14.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.419.0 to 2.420.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.419.0 to 2.420.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.419.0...v2.420.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17725 from emberjs/dependabot/npm_and_yarn/tslint-5.14.0

dependabot[bot] committed:
Merge pull request #17726 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.420.0

rwjblue committed:
Merge pull request #17719 from emberjs/angle-bracket-nested-lookups

Support for nested components in angle bracket invocation syntax

Commit:
[DOC canary] Fixup docs for @action.

* Fix copy/pasta error in method name
* Add `@category` to feature flag the docs

Commit:
[DOC release] Remove documentation feature flagging for enabled features

Commit:
[DOC canary] Fix @category annotation for buildRouteInfoMetadata.

chancancode committed:
[CLEANUP] remove unflagged features

chancancode committed:
Merge pull request #17729 from emberjs/feature-cleanup

[CLEANUP] remove unflagged features

chancancode committed:
Add ember-glimmer-angle-bracket-nested-lookup to FEATURES.md

chancancode committed:
Move helper registration tests

chancancode committed:
Move link-to tests

chancancode committed:
Merge pull request #17730 from emberjs/refactor-helper-tests

Move link-to/helper tests into glimmer tests folder

Commit:
[BUGFIX] Adds a warning to {{action}} for undecorated methods

This PR adds a warning to the `{{action}}` modifier and the `{{action}}`
helper which tells the user they should add the `@action` decorator to
any method which is passed to it. We're only warning for the time being
since this _is_ a supported codepath, but has been actively discouraged
in the past.

simonihmig committed:
Fix initializer blueprint for ember-mocha 0.14

The previous PR #17411 and its last-minute change to remove `setupTest()` introduced a very stupid error! 🤦‍♂️

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Implement `<Textarea />`

chancancode committed:
[BUGFIX beta] assert `<Input>` and `<Textarea>`

`input` and `textarea` are intended to be reserved component names, but
the assertion was added to the wrong place. The end result is that only
`(component "input")` and `(component "textarea")` were disallowed, but
not when using the angle bracket syntax `<Input />` and `<Textarea />`.

This compliments the existing tests in `contextual-components-test.js`
(L1220).

chancancode committed:
[CLEANUP] more feature cleanup

rwjblue committed:
Merge pull request #17733 from emberjs/built-in-asserts

[BUGFIX beta] assert `<Input>` and `<Textarea>`

rwjblue committed:
Merge pull request #17734 from emberjs/more-feature-cleanup

[CLEANUP] more feature cleanup

rwjblue committed:
Merge pull request #17732 from simonihmig/fix-initializer-blueprint

Fix initializer blueprint for ember-mocha 0.14

rwjblue committed:
Merge pull request #17728 from rwjblue/fix-docs-for-action-decorator

[DOC canary] Fixup docs for @action.

dependabot-support committed:
Bump aws-sdk from 2.420.0 to 2.421.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.420.0 to 2.421.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.420.0...v2.421.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17736 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.421.0

Commit:
Remove EMBER_ROUTING_ROUTER_SERVICE feature flagging.

Commit:
Remove EMBER_ENGINES_MOUNT_PARAMS feature.

rwjblue committed:
Merge pull request #17739 from rwjblue/fixup-remove-older-features

More "enabled feature" cleanup.

ppcano committed:
Fix octane component blueprint and octane blueprint tests

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Remove `{{dash-rule}}` for curly invocations.

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Implement `<TextArea />` typo guard.

rwjblue committed:
Merge pull request #17740 from ppcano/fix-octane-blueprint-tests

Fix octane component blueprint and octane blueprint tests

dependabot-support committed:
Bump ember-cli-babel from 7.5.0 to 7.6.0

Bumps [ember-cli-babel](https://github.com/babel/ember-cli-babel) from 7.5.0 to 7.6.0.
- [Release notes](https://github.com/babel/ember-cli-babel/releases)
- [Changelog](https://github.com/babel/ember-cli-babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/ember-cli-babel/compare/v7.5.0...v7.6.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17741 from emberjs/dependabot/npm_and_yarn/ember-cli-babel-7.6.0

dependabot-support committed:
Bump aws-sdk from 2.421.0 to 2.422.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.421.0 to 2.422.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.421.0...v2.422.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17742 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.422.0

chancancode committed:
[FEATURE angle-bracket-nested-lookup] enable 👌

Commit:
fix TSLint

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Implement `<Input />`.

simonihmig committed:
Fix component-test blueprint for ember-cli-mocha

Removes incidental use of QUnit's `assert()` introduced in #17685.

rwjblue committed:
Merge pull request #17731 from emberjs/octane/add-warning-to-action-helper

[BUGFIX] Adds a warning to {{action}} for undecorated methods

rwjblue committed:
Merge pull request #17747 from simonihmig/fix-mocha-component-blueprint

Fix component-test blueprint for ember-cli-mocha

chancancode committed:
[CLEANUP beta] remove unused transforms

`ComponentNode` has not been a thing for a long time.

chancancode committed:
[WIP] API docs for nested angle invocation

chancancode committed:
Merge pull request #17748 from emberjs/cleanup-transforms

[CLEANUP beta] remove unused transforms

wycats committed:
A number of improvements to component docs

- Removed legacy langauge about "view"s
- Removed legacy language about "context"s and generally
  avoid unnecessarily referring to the controller as an
  important concept involved in {{yield}}
- Added headers
- Clean up language around the dash rule (to be fully removed
  in a near-term commit)
- Clean up language about the `layout` property that was too
  general
- Remove reference to jQuery.Event
- Other assorted aesthetic cleanups

simonihmig committed:
Fix mixin blueprint tests under Windows

Uncovered in https://github.com/typed-ember/ember-cli-typescript-blueprints/pull/74, tests fail under Windows due to hardcoded line ending character.

rwjblue committed:
Merge pull request #17750 from simonihmig/mixin-blueprint-eol

Fix mixin blueprint tests under Windows

chancancode committed:
Merge pull request #17745 from emberjs/enable-nested-angle

[FEATURE ember-glimmer-angle-bracket-nested-lookup] enable 👌

locks committed:
Update component.ts

simonihmig committed:
initializer blueprint: removed unused import

`setupTest()` was removed, but the import was left behind. (Dang, sorry!)

chancancode committed:
cleanup {{link-to}} tests

rwjblue committed:
Update packages/@ember/-internals/glimmer/lib/component.ts

Co-Authored-By: wycats <wycats@gmail.com>

chancancode committed:
Merge pull request #17753 from emberjs/cleanup-link-to-tests

cleanup `{{link-to}}` tests

dependabot-support committed:
Bump eslint from 5.15.1 to 5.15.2

Bumps [eslint](https://github.com/eslint/eslint) from 5.15.1 to 5.15.2.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v5.15.1...v5.15.2)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17760 from emberjs/dependabot/npm_and_yarn/eslint-5.15.2

Commit:
[BUGFIX canary] Ensure uninitialized tracked properties "work".

DingoEatingFuzz committed:
Add examples to the urlFor RouterService method

habdelra committed:
Fixes 595 in ember-learn/ember-api-docs - missing has-block API documentation

locks committed:
upgrade ember-cli-yuidoc to address vulnerability warnings

Commit:
Adds test to confirm that GH#17018 is not an issue

habdelra committed:
prettier lint fix

rwjblue committed:
Merge pull request #17765 from rwjblue/ensure-initializer-is-function

[BUGFIX canary] Ensure uninitialized tracked properties "work".

habdelra committed:
Fixes 594 in ember-learn/ember-api-docs - add has-block-params API documentation

chancancode committed:
Merge pull request #17769 from shawnren/test-passing-model-to-link-to-via-hash

Adds test to confirm that GH#17018 is not an issue

rwjblue committed:
Merge pull request #17766 from DingoEatingFuzz/document-router-service-methods

Add examples to the urlFor RouterService method

Turbo87 committed:
Merge pull request #17752 from simonihmig/init-blueprint-unused-import

initializer blueprint: removed unused import

chancancode committed:
Merge pull request #17754 from emberjs/enable-nested-angle

Update component documentation

burieberry committed:
[DOCS] Add code example for router service `location` property

Part of #16518

rwjblue committed:
Merge branch 'master' into angle-built-ins

rwjblue committed:
Merge pull request #17735 from emberjs/angle-built-ins

[FEATURE ember-glimmer-angle-bracket-built-ins]

Commit:
Fix linting failure.

habdelra committed:
added param "name of the block" for the API document of has-block and has-block-params

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Implement `<LinkTo />`.

chancancode committed:
Revert "[BUGFIX] Adds a warning to {{action}} for undecorated methods"

chancancode committed:
Enable Octane features in alpha builds

This makes it easier to try Octane in some non-standard workflow
(and benchmarks, etc). Also, this would allow the Octane blueprint
to start tracking the alpha channel instead of canary, keeping the
list of Octane feature flags in-sync more easily.

chancancode committed:
Merge pull request #17773 from emberjs/revert-17731-octane/add-warning-to-action-helper

Revert "[BUGFIX] Adds a warning to {{action}} for undecorated methods"

chancancode committed:
Fix docs issue

chancancode committed:
Merge pull request #17774 from emberjs/octane-alpha

Enable Octane features in alpha builds

dependabot-support committed:
Bump aws-sdk from 2.422.0 to 2.423.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.422.0 to 2.423.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.422.0...v2.423.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump eslint from 5.15.2 to 5.15.3

Bumps [eslint](https://github.com/eslint/eslint) from 5.15.2 to 5.15.3.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v5.15.2...v5.15.3)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17776 from emberjs/dependabot/npm_and_yarn/eslint-5.15.3

dependabot[bot] committed:
Merge pull request #17775 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.423.0

rwjblue committed:
Merge pull request #17772 from emberjs/angle-link-to

[FEATURE ember-glimmer-angle-bracket-built-ins]

chancancode committed:
refactor link-to transform tests

chancancode committed:
Merge pull request #17777 from emberjs/angle-link-to

refactor link-to transform tests

chancancode committed:
[CLEANUP] remove `-html-safe` helper

Since #17777, nothing uses this private helper anymore

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Transform positional `{{link-to}}` invocations into named arguments.
The runtime path still need to be kept around for custom subclasses.

dependabot-support committed:
Bump @babel/plugin-transform-block-scoping from 7.3.4 to 7.4.0

Bumps [@babel/plugin-transform-block-scoping](https://github.com/babel/babel) from 7.3.4 to 7.4.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/compare/v7.3.4...v7.4.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.423.0 to 2.424.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.423.0 to 2.424.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.423.0...v2.424.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump @babel/plugin-transform-parameters from 7.3.3 to 7.4.0

Bumps [@babel/plugin-transform-parameters](https://github.com/babel/babel) from 7.3.3 to 7.4.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/compare/v7.3.3...v7.4.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17781 from emberjs/dependabot/npm_and_yarn/@babel/plugin-transform-block-scoping-7.4.0

dependabot-support committed:
Bump @babel/plugin-transform-classes from 7.3.4 to 7.4.0

Bumps [@babel/plugin-transform-classes](https://github.com/babel/babel) from 7.3.4 to 7.4.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/compare/v7.3.4...v7.4.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump @babel/plugin-proposal-class-properties from 7.3.4 to 7.4.0

Bumps [@babel/plugin-proposal-class-properties](https://github.com/babel/babel) from 7.3.4 to 7.4.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/compare/v7.3.4...v7.4.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17784 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.424.0

dependabot[bot] committed:
Merge pull request #17786 from emberjs/dependabot/npm_and_yarn/@babel/plugin-transform-parameters-7.4.0

dependabot[bot] committed:
Merge pull request #17783 from emberjs/dependabot/npm_and_yarn/@babel/plugin-transform-classes-7.4.0

dependabot[bot] committed:
Merge pull request #17782 from emberjs/dependabot/npm_and_yarn/@babel/plugin-proposal-class-properties-7.4.0

dependabot-support committed:
Bump @babel/plugin-transform-destructuring from 7.3.2 to 7.4.0

Bumps [@babel/plugin-transform-destructuring](https://github.com/babel/babel) from 7.3.2 to 7.4.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/compare/v7.3.2...v7.4.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17787 from emberjs/dependabot/npm_and_yarn/@babel/plugin-transform-destructuring-7.4.0

wagenet committed:
[BUGFIX] Tests and support for native DOM events on Inputs

rwjblue committed:
Merge pull request #17779 from emberjs/angle-link-to

Transform positional `{{link-to}}` invocations into named arguments.

dependabot-support committed:
Bump aws-sdk from 2.424.0 to 2.425.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.424.0 to 2.425.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.424.0...v2.425.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17789 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.425.0

rwjblue committed:
Merge pull request #17778 from emberjs/rm-html-safe

[CLEANUP] remove `-html-safe` helper

rwjblue committed:
Merge pull request #17768 from emberjs/v11y-dependencies

upgrade ember-cli-yuidoc to address vulnerability warnings

rwjblue committed:
Merge pull request #17767 from habdelra/has-block-api-documentation

Fixes 595 in ember-learn/ember-api-docs - missing has-block API documentation

dependabot-support committed:
Bump aws-sdk from 2.425.0 to 2.426.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.425.0 to 2.426.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.425.0...v2.426.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17790 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.426.0

wagenet committed:
Merge pull request #17788 from wagenet/failing-angle-input-test

Fix native DOM events in Glimmer Angle Brackets

dependabot-support committed:
Bump broccoli-uglify-sourcemap from 3.0.0 to 3.1.0

Bumps [broccoli-uglify-sourcemap](https://github.com/ember-cli/broccoli-uglify-sourcemap) from 3.0.0 to 3.1.0.
- [Release notes](https://github.com/ember-cli/broccoli-uglify-sourcemap/releases)
- [Changelog](https://github.com/ember-cli/broccoli-uglify-sourcemap/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ember-cli/broccoli-uglify-sourcemap/compare/v3.0.0...v3.1.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.426.0 to 2.427.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.426.0 to 2.427.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.426.0...v2.427.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17794 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.427.0

chancancode committed:
disable unused CAPABILITIES

tansongyang committed:
[DOC RouteInfo] Fix grammar, spelling, and formatting

dependabot[bot] committed:
Merge pull request #17793 from emberjs/dependabot/npm_and_yarn/broccoli-uglify-sourcemap-3.1.0

rwjblue committed:
Merge pull request #17797 from tansongyang/patch-1

[DOC RouteInfo] Fix grammar, spelling, and formatting

kategengler committed:
Add v3.9.0-beta.5 to CHANGELOG
[ci skip]

(cherry picked from commit 1e2672cddfd7bc277c8073472635b1b6ab111200)

dependabot-support committed:
Bump aws-sdk from 2.427.0 to 2.428.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.427.0 to 2.428.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.427.0...v2.428.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump rollup-plugin-commonjs from 9.2.1 to 9.2.2

Bumps [rollup-plugin-commonjs](https://github.com/rollup/rollup-plugin-commonjs) from 9.2.1 to 9.2.2.
- [Release notes](https://github.com/rollup/rollup-plugin-commonjs/releases)
- [Changelog](https://github.com/rollup/rollup-plugin-commonjs/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup-plugin-commonjs/compare/v9.2.1...v9.2.2)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17800 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.428.0

dependabot[bot] committed:
Merge pull request #17801 from emberjs/dependabot/npm_and_yarn/rollup-plugin-commonjs-9.2.2

chancancode committed:
Merge pull request #17798 from emberjs/input-manager

Template-only components: disable unused CAPABILITIES

tansongyang committed:
[DOC RouteInfo] Fix `find` docs

rwjblue committed:
Merge pull request #17802 from tansongyang/patch-1

[DOC RouteInfo] Fix `find` docs

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Refactor #17788 to internally forward all the args (essentially `...@args`)
in the component manager layer instead.

chancancode committed:
Don't register `Checkbox` and `TextField`

This didn't work as intended, as they end up being invokable with `{{checkbox}}`
and `{{textfield}}` (the intention was to keep they private).

locks commented:
Can you update this PR? 🙏

locks committed:
Merge pull request #17539 from IzzatN/cleanup-packages

removed duplicate line

chancancode committed:
[CLEANUP] remove unused `findElementInParentElement`

locks committed:
Merge pull request #16798 from delusioninabox/docs/clarify-unless

[DOC] Improve documentation of unless template helper (issue #16592)

rwjblue committed:
Merge pull request #17803 from emberjs/input-manager

Refactor #17788 to use ...@args

chancancode committed:
[BUGFIX canary] don't rely on `elementId` for event dispatching

Since the `id` attribute can be passed through splattributes, relying
on `elementId` for event dispatching is not quite correct.

locks committed:
Merge pull request #16966 from bekzod/cleanup-route

cleanup route

dependabot-support committed:
Bump ember-cli-version-checker from 3.1.2 to 3.1.3

Bumps [ember-cli-version-checker](https://github.com/rwjblue/ember-cli-version-checker) from 3.1.2 to 3.1.3.
- [Release notes](https://github.com/rwjblue/ember-cli-version-checker/releases)
- [Changelog](https://github.com/ember-cli/ember-cli-version-checker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rwjblue/ember-cli-version-checker/compare/v3.1.2...v3.1.3)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.428.0 to 2.429.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.428.0 to 2.429.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.428.0...v2.429.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Merge pull request #17806 from emberjs/dependabot/npm_and_yarn/ember-cli-version-checker-3.1.3

dependabot[bot] committed:
Merge pull request #17807 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.429.0

locks committed:
Merge pull request #17429 from mike-north/shellcheck

bin/publish_builds cleanup

burieberry committed:
Requested changes

chancancode committed:
stop using @id in tests

chancancode committed:
export getElementView

wycats committed:
Update docs for the builtin components

wycats committed:
Fix expected lint

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins] enable 👌

dependabot-support committed:
Bump ember-cli-babel from 7.6.0 to 7.7.0

Bumps [ember-cli-babel](https://github.com/babel/ember-cli-babel) from 7.6.0 to 7.7.0.
- [Release notes](https://github.com/babel/ember-cli-babel/releases)
- [Changelog](https://github.com/babel/ember-cli-babel/blob/master/CHANGELOG.md)
- [Commits](https://github.com/babel/ember-cli-babel/compare/v7.6.0...v7.7.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump aws-sdk from 2.429.0 to 2.430.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.429.0 to 2.430.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.429.0...v2.430.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Add missing dependent key

There are no failing tests for this at the moment, because we only
test with all optional features on or off. It turns out that the
tracked property feature is masking this failure. As a result, the
PR that enables this feature (#17811) without tracked property has
some failing tests which should be green again after rebasing on
top of this patch.

dependabot[bot] committed:
Merge pull request #17812 from emberjs/dependabot/npm_and_yarn/ember-cli-babel-7.7.0

dependabot[bot] committed:
Merge pull request #17813 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.430.0

wycats committed:
Address @chancancode's issues

chancancode committed:
Merge pull request #17814 from emberjs/missing-href-dep-key

[FEATURE ember-glimmer-angle-bracket-built-ins] Add missing dependent key

chancancode committed:
Merge pull request #17818 from emberjs/fix-event-dispatcher-1

Fix event dispatcher (part 1)

dependabot-support committed:
Bump aws-sdk from 2.430.0 to 2.431.0

Bumps [aws-sdk](https://github.com/aws/aws-sdk-js) from 2.430.0 to 2.431.0.
- [Release notes](https://github.com/aws/aws-sdk-js/releases)
- [Changelog](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-js/compare/v2.430.0...v2.431.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot-support committed:
Bump puppeteer from 1.13.0 to 1.14.0

Bumps [puppeteer](https://github.com/GoogleChrome/puppeteer) from 1.13.0 to 1.14.0.
- [Release notes](https://github.com/GoogleChrome/puppeteer/releases)
- [Commits](https://github.com/GoogleChrome/puppeteer/compare/v1.13.0...v1.14.0)

Signed-off-by: dependabot[bot] <support@dependabot.com>

chadhietala committed:
Bump router_js

chadhietala committed:
Merge pull request #17823 from emberjs/bump-router-js

Bump router_js

chancancode committed:
[FEATURE ember-glimmer-angle-bracket-built-ins]

Fix events test

chancancode committed:
Merge pull request #17810 from emberjs/update-builtin-component-docs

Update docs for the builtin components

chancancode committed:
Merge pull request #17811 from emberjs/enable-builtin-angles

[FEATURE ember-glimmer-angle-bracket-built-ins] enable 👌

locks commented:
@lifeart hi! do you have some time to handle the conflicts?

dependabot[bot] committed:
Merge pull request #17821 from emberjs/dependabot/npm_and_yarn/aws-sdk-2.431.0

dependabot[bot] committed:
Merge pull request #17822 from emberjs/dependabot/npm_and_yarn/puppeteer-1.14.0

locks committed:
Merge pull request #17770 from burieberry/router-location-docs

[DOCS] Add code example for router service `location` property

chancancode committed:
Upgrade Glimmer and TypeScript

@glimmer/* 0.37.1 -> 0.38.1
typescript 3.0.1 -> 3.2.4

chancancode committed:
Add missing babel helper

This is needed due to https://github.com/glimmerjs/glimmer-vm/blob/e672233ea70ab2614648bb194efc107f3698baad/packages/%40glimmer/runtime/lib/compiled/opcodes/component.ts

This is probably unintended and a waste of bytes. We should fix it
upstream and remove this helper.

chancancode committed:
Merge pull request #17824 from emberjs/glimmer-0-38

Upgrade Glimmer and TypeScript

lifeart commented:
@locks yeah, I'm in

lifeart committed:
Merge branch 'patch-1' of https://github.com/lifeart/ember.js into patch-1

lifeart commented:
I will open another PR )
